library/src/main/java/io/andref/rx/widgets/ListViewCard.java

```diff
@@ -29,23 +29,25 @@
 
 import rx.Observable;
 import rx.functions.Func1;
+import rx.subjects.PublishSubject;
 
 public class ListViewCard extends FrameLayout
 {
     private static final String TAG = "ListViewCard";
 
     private List<Item> mItems = new ArrayList<>();
 
+    private final PublishSubject<Item> mItemClicks = PublishSubject.create();
+    private final PublishSubject<Item> mIconClicks = PublishSubject.create();
+
     private Observable<Void> mButtonClicks;
```

Comment:
After removing all views and adding another, none of the clicks on the newly added views are being recognized